documentation/en/separate-daemon-miner.md

```diff
@@ -0,0 +1,28 @@
+# Running Lotus Daemon and Lotus-Miner on separate hosts
+
+To setup the lotus daemon and lotus-miner on separate hosts, first setup up the daemon on your chosen machine. Make sure that the lotus daemon's API is configured to be accessible by an external host.
+
+```sh
+[API]
+  ListenAddress = "/ip4/0.0.0.0/tcp/1234/http"
+```
+
+Once the lotus daemon is configured and running, the lotus miner host needs to be configured. On the lotus miner machine, create a ~/.lotus directory like on the lotus daemon machine but only create the ~/.lotus/api and ~/.lotus/token files. Make sure that the ~/.lotus/api files contains the appropriate multiaddress for the machine that is running the lotus daemon, i.e.
+
+```sh 
+$ cat ~/.lotus/api
+/ip4/192.168.1.11/tcp/1234/http
+```
+
+Note, this should differ from the contents of the file on the lotus daemon machine.
+
+For the token file, you can copy the contents of the ~/.lotus/token file on the lotus daemon machine exactly.
+
+So on the miner machine you end up with a ~/.lotus directory that looks like the following:
+
+```sh
+$ ls ~/.lotus
+-rw-r--r-- 1 lotus lotus  32 Aug 25 15:49 api
+-rw-r--r-- 1 lotus lotus 137 Aug 25 15:49 token
+```
```

Comment:
I guess this is poorly documented - but you can use the `FULLNODE_API_INFO` env var for this instead of creating fake lotus repo
```
lotus auth api-info --perm admin
FULLNODE_API_INFO=eyJhbGci[...]4HVg:/ip4/127.0.0.1/tcp/1234/http

```

Reply:
In FULLNODE_API_INFO I believe there should be set an IP address of host where the daemon is running, right?
I ran into some strange behaviour while trying to get miner to work with a remote daemon. #4209